### DISCUSSION TITLE
Why the XSPH algorithm moving from viscosity to the standalone nonPressure force?

### DISCUSSION THREAD

**Discussion by chunleili** [InteractiveComputerGraphics/SPlisHSPlasH#230] [Q&A]
Hello,
  I am curious why the XSPH moving from viscosity to a standalone non-pressure force algorithm in the 2.12 version? In my point of view, XSPH is viscosity algorithm and it seems approperiate to place it in the viscosity algorithm as before.

**Comment by janbender** [comment]
XSPH was originally introduced as velocity filter which is somehow required in simulations with very low viscosity to smooth the velocity field. Otherwise you get completely chaotic motion. Some researchers asked for simulating inviscid or almost inviscid fluids. Now it is kind of misleading that XSPH is sold as viscosity method since you need it in such simulations. If you are interested, maybe you could take a look at the works of Monaghan to get more info.
